src/docfx/config/ops/OpsPreProcessor.cs

```diff
@@ -71,7 +71,7 @@ private void LogError(LogItem item)
         {
             if (!string.IsNullOrEmpty(item.Code))
             {
-                _errorLog.Write(new Error(MapLevel(item.MessageSeverity), item.Code, item.Message, new FilePath(item.File), item.Line ?? 0));
+                _errorLog.Write(new Error(MapLevel(item.MessageSeverity), item.Code, item.Message, new FilePath(item.File ?? ""), item.Line ?? 0));
```

Comment:
consider give `Error` a `null` `FilePath` in this case. From the type declaration, you can see that `Error.FilePath` is nullable, which indicate it is using `null` there to represent an error without a clear source file.